lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -0,0 +1,183 @@
+#!powershell
+#
+# Copyright: (c) 2017, AMTEGA - Xunta de Galicia
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+
+# ------------------------------------------------------------------------------
+$ErrorActionPreference = "Stop"
+
+# Preparing result
+$result = @{}
+Set-Attr $result "changed" $false
```

Comment:
Avoid using the `Set-Attr` cmdlets as they were only necessary when we used psobjects for the result. Instead you could do this in 2 different ways

```
$result = @{
    changed = $false
}

# or id result is already defined and you adding/changing a ney key
$result = @{}
$result.changed = $false
```